packages/backend/src/services/CommentService/CommentService.ts

```diff
@@ -145,6 +155,13 @@ export class CommentService {
                 "You don't have access to the space this dashboard belongs to",
             );
         }
+        analytics.track({
+            event: 'comment.resolved',
+            userId: user.userUuid,
+            properties: {
+                // TODO if you want to add properties, we need a getCommentById method (we can refactor getCommentOwner)
```

Comment:
good point, like a 
`isOwner` or something?

Reply:
I was thinking refactor the `getCommentOwner` to return the whole comment object, to make it more generic,  and check userUuid in the service itself